spec/ic.did

```diff
@@ -115,8 +115,9 @@ service ic : {
     method : variant { get; head; post };
     headers: vec http_header;
     body : opt blob;
+    transform_context: blob;
     transform : opt variant {
-      function: func (http_response) -> (http_response) query
+      function: func (http_response, blob) -> (http_response) query
```

Comment:
For consistency, should the two argument sequence `(http_repsonse, blob)`  be packed into a record `(response: http_response; context: blob)`? 

This is legal Candid as is, but everywhere else in the IC management canister interface uses a single record of arguments and results, never multiple arguments.

Reply:
```suggestion
      context : blob;
      function: func { response : http_response; context:  blob} -> (http_response) query
```
Since the transform itself is optional, perhaps the context should only be specified as part of the variant. Why require a context when no transform is provided? This way, you supply neither of both.

(and delete transform_context at l. 118)